Dockerfile

```diff
@@ -10,6 +11,26 @@ RUN ./node_modules/.bin/gulp build
 RUN npm run lint:js -- --max-warnings=0
 RUN npm run lint:css
 
+FROM debian as react-builder
+
+# Install Volta, to use the same toolchain (Node+npm) as developers locally:
+RUN apt-get update && apt-get install -y curl ca-certificates --no-install-recommends
+ENV VOLTA_HOME $HOME/.volta
+ENV PATH $VOLTA_HOME/bin:$PATH
+RUN curl https://get.volta.sh | bash
+
+WORKDIR /app
+COPY privaterelay/locales ./privaterelay/locales
+# TODO: Move static files from /static/ to /frontend/public/ and remove this COPY:
```

Comment:
This TODO is for _after_ we remove the old frontend from the codebase, since that's still referring to files in `/static/`, of course.